src/main/js/controls/Graph/Graph.js

```diff
@@ -418,7 +418,10 @@ class Graph extends Construct {
                 if (key === graphData.key) position = index;
             });
             if (position >= 0) {
-                if (this.content[position].type === "Bar") {
+                if (
+                    this.content[position].type === "Bar" &&
+                    graphData.values.length > 0
```

Comment:
what is the value of this change?  are we trying to prevent `this.config.axis.x.ticks.values = [];` from executing when the length of the `graphData.values.length` is zero?

Reply:
Yes Andy you are if we do not do so, we are getting an error saying that tick values are empty.